modules/setup_unix/environment_installation.md

```diff
@@ -372,46 +364,33 @@ git --version
 
 which should show something like `git version 2.7.0`.
 
-### editor (atom)
+### editor (vs code)
 
 #### First time
 
-Open `atom` using your GUI
+Open `vs code` using your GUI
 * Windows: from the start menu
 * macOS: from the Application folder
 * Linux: varies (but you might be able to skip to "From the shell"
 
-A window should open showing the atom logo and welcome screen, similar
+A window should open showing the vs code logo and get started screen, similar
 to 
 
-![atom welcome screenshot]({{ site.baseurl }}/{{ site.figs }}/atom_welcome.jpg)
+![vs code get started screenshot]({{ site.baseurl }}/{{ site.figs }}/vscode_get_started.jpg)
 
-If it tries to install additional commands (`atom` and `apm`) then let
-it do it and provide your system administrator password if required.
-
-Then exit the editor again (Quit from the menu or close the
+Then exit the editor again (Exit from the menu or close the
 window).
 
 #### From the shell
 
 In the shell, type
 
 ```bash
-atom
+code
 ```
 
 It should open the editor. Exit the editor.
 
-If this does not work then you need to let atom install additional
-commands. The Atom menu bar has a command named *"Install Shell
-Commands"* which installs the atom and apm commands, as described
-under [Opening a
-File](https://flight-manual.atom.io/getting-started/sections/atom-basics/#opening-a-file)
-(and provide your system administrator password if requested). Then
-add the atom directory to your bash start file manually as described
-under Wiki:[start atom from the git-bash
-commandline]({{ site.wiki.url }}/installation-troubleshooting#start-atom-from-the-git-bash-commandline).)
-
```

Comment:
On macOS, additional commands need to be installed, see https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line.

On Linux and Windows, the `code` command should be available if you followed the standard installation instructions.